orc8r/cloud/go/syncstore/store_test.go

```diff
@@ -0,0 +1,333 @@
+/*
+ Copyright 2020 The Magma Authors.
+
+ This source code is licensed under the BSD-style license found in the
+ LICENSE file in the root directory of this source tree.
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package syncstore_test
+
+import (
+	"testing"
+	"time"
+
+	"magma/orc8r/cloud/go/blobstore"
+	"magma/orc8r/cloud/go/clock"
+	configurator_storage "magma/orc8r/cloud/go/services/configurator/storage"
+	"magma/orc8r/cloud/go/sqorc"
+	"magma/orc8r/cloud/go/syncstore"
+	"magma/orc8r/lib/go/protos"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSyncStore(t *testing.T) {
+	db, err := sqorc.Open("sqlite3", ":memory:")
+	assert.NoError(t, err)
+	fact := blobstore.NewSQLBlobStorageFactory("last_resync_time", db, sqorc.GetSqlBuilder())
```

Comment:
Here and elsewhere: use this instead https://github.com/magma/magma/blob/1b784496f3577c020f13236e815488cd8d9ee670/orc8r/cloud/go/test_utils/test_db.go#L49-L51